### ISSUE TITLE
[Feature request]  Reference image

### ISSUE THREAD

**Issue by mdtrooper** [guillaumechereau/goxel#231]
Hi.

I thought that there was this feature in Goxel.

I don't the technical name, but there is something similar in Blender.

For example in this video:

https://www.youtube.com/watch?v=-49fRP1Y5TI

It is to do a 3D from old game sprites.

Regards.

**Comment by mdtrooper** [comment]
Sorry I found it. But there are somethings to improve:

- when import image plane, maybe the img layer could be the name of image file, and icon of image, and for voxel layer maybe could be other icon.
- in the tool palette, the move or rotation doesn't change the value (they show 0 value).
- maybe good idea the move will be 1px and 1px will be 1 voxel.
- showing the name of selected layer is necesary in some place of window.
- there are some minor graphical glitches (but they could be my intel graphics card in GNU/Linux).

![Captura de pantalla -2021-02-28 03-10-27](https://user-images.githubusercontent.com/1285451/109405635-ad8f1480-7972-11eb-826d-9311986c34ef.png)

But, thanks, Goxel is awesome.

**Comment by guillaumechereau** [comment]
Yes this is feature I added quickly and didn't really worked on.  Some improvements are indeed needed.

**Comment by mdtrooper** [comment]
> Yes this is feature I added quickly and didn't really worked on. Some improvements are indeed needed.

Ohh I understand you. It is ok.
